### PR TITLE
Add ELB surge queue and spillover count panels

### PR DESCRIPTION
**Pull request by peggyl**

### CODE REVIEW

File: dashboards/scripted_services.js [hunk 1]
```diff
@@ -404,6 +404,64 @@ return function(callback) {
...
+      "type": "graph",
+      "datasource": "CloudWatch",
+      "targets": [
+        {
+          "function": "mean",
+          "column": "value",
+          "namespace": "AWS/ELB",
+          "metricName": "SurgeQueueLength",
+          "statistics": [
+            "Average",
```
Comment (mohit) [inline dashboards/scripted_services.js]: use `Max`. See `SurgeQueueLength` Statistics http://docs.aws.amazon.com/ElasticLoadBalancing/latest/DeveloperGuide/elb-cloudwatch-metrics.html

File: dashboards/scripted_services.js [hunk 2]
```diff
@@ -404,6 +404,64 @@ return function(callback) {
...
+      "type": "graph",
+      "datasource": "CloudWatch",
+      "targets": [
+        {
+          "function": "mean",
+          "column": "value",
+          "namespace": "AWS/ELB",
+          "metricName": "SpilloverCount",
+          "statistics": [
+            "Average",
```
Comment (mohit) [inline dashboards/scripted_services.js]: Use `average`

**Comment by mohit** [comment]
Is there a way for us to know if there are no elbs associated with a service? In that case it would be nice to remove the ELB related graphs. Does not need to be a part of this PR.

**Comment by mohit** [comment]
needs update of the statistics used for metrics. apart from that LGTM

File: dashboards/scripted_services.js (hunk 2 above)
Comment (peggyl) [inline dashboards/scripted_services.js]: Does it need to be lowercase? I've used uppercase `Average` with other graphs and that's worked fine.

Comment (peggyl) [inline dashboards/scripted_services.js]: Docs recommend `Sum` - assuming that's what you meant (instead of `Average`)?

Comment (mohit) [inline dashboards/scripted_services.js]: Sorry, you're right I meant `Sum`

**Comment by mohit** [comment]
LGTM

**Comment by peggyl** [comment]
@nathanleiby Just to sanity-check, can we reliably assume ecs-metrics will omit the `elb_name` key or have an empty string for the value in a Service's JSON representation if an ELB doesn't exist for that service?

**Comment by peggyl** [comment]
@mohit I'll merge this next time I'm on a laptop and can reliably deploy it too. The check-and-skip ELB charts is a good idea and very quick to do; I will definitely add that soon.

**Pull request merged**